massa-bootstrap/src/server.rs

```diff
@@ -63,11 +63,15 @@ use crate::{
     white_black_list::SharedWhiteBlackList,
     BootstrapConfig,
 };
+
+/*
```

Comment:
Why is this commented? If we need it for reference, add a comment explaining it, otherwise remove.

Reply:
Removed